puppetboard/static/js/environments.js

```diff
@@ -0,0 +1,11 @@
+(function () {
+  var $;
+
+  $ = jQuery;
+
+  $('#switch_env').change(function() {
+    path = location.pathname.split('/');
+    path[1] = $(this).find(':selected').text();
+    location.assign(path.join('/'))
```

Comment:
I prefer a bootstrap-like dropdown in the navbar with the hyperlinks directly setup. Am planning on (re-)implementing bootstrap after the 0.1.0 release.